test/unit/validation/string.spec.js

```diff
@@ -443,5 +443,38 @@ describe('validation', () => {
         });
       });
     });
+
+    describe('uuid', () => {
+      var User;
```

Comment:
You could define your schema directly here, what you think?
```js
const User = attributes({
  id: {
    type: String,
    uuid: true
  }
})(class User {});
```

Reply:
I would keep it as it is so if we use `.only` somewhere else (or use `.skip` here) it would work as expected, but if we change it it would still run the decorator.

Reply:
Sure, got it.